AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -213,13 +213,13 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
         Report.result(Tests.viewport_background_selected, neutral_urban_background_loaded is True)
 
         # 11. Verify the lighting background asset is the expected value.
-        # neutral_urban_background_path = os.path.join(
-        #     azlmbr.paths.engroot, "Gems", "Atom", "Tools", "MaterialEditor", "Assets", "MaterialEditor",
-        #     "LightingPresets", "neutral_urban.lightingpreset.azasset")
-        # neutral_urban_background_path = neutral_urban_background_path.replace(os.sep, '/')
-        # Report.result(
-        #     Tests.viewport_background_has_expected_asset,
-        #     atom_tools_utils.get_last_lighting_preset_path() == neutral_urban_background_path)
+        neutral_urban_background_path = os.path.join(
```

Comment:
Do you make similar paths in other tests? Maybe it would it be worth making this into a function in a tools file someplace? I'm seeing `"Gems", "Atom", "Tools", "MaterialEditor", "Assets", "MaterialEditor"` multiple times here.

Reply:
forgot to add to this also, it got merged before i was able to update (guthrie merged it for me)

in the next update when we add MaterialCanvas tests, this will get changed - probably as a constant instead of a function though at the top of the `atom_tools_utils.py` module

ill link it here when i open that PR

Reply:
here it is: https://github.com/o3de/o3de/pull/13509